handbook/security.md

```diff
@@ -0,0 +1,265 @@
+* Using the firewall will ensure that we limit the exposure to our devices, while Stealth mode makes them more difficult to discover. 
+* Firewall logging allows us to troubleshoot and investigate whether the firewall blocks applications or connections.
+
+**User experience impact**
+
+* Due to FileVault's encryption process, a password is needed as soon as the laptop is turned on, instead of once it has booted.
+* No performance impact - macOS encrypts the system drive by default. 
+* With Gatekeeper enabled, unsigned or unnotarized (not checked for malware by Apple) applications require extra steps to execute.
+* With the firewall enabled, unsigned applications cannot open a firewall port for inbound connections.
+
+#### Screen saver and automatic locking
+
+| #     | Setting                                                                             |
+| ----- | ----------------------------------------------------------------------------------- |
+| 2.3.1 | Ensure an inactivity interval of 20 minutes or less for the screen saver is enabled |
+| 6.1.2 | Ensure show password hint is disabled                                              |
+| 6.1.3 | Ensure guest account is disabled                                                    |
+| NA    | Prevent the use of automatic logon                                                  |
+
+**Why?**
+
+* Fleet employees are free to work from wherever they choose. If a laptop is lost or forgotten, automatic login exposes sensitive company data and poses a critical security risk. 
+* Password hints can sometimes be easier to guess than the password itself. Since we support employees remotely via MDM and do not require users to change passwords frequently, we eliminate the need for passwords hints and their associated risk.
+* Only a single employee should use a company laptop. Therefore, guest accounts are not permitted.
+* Automatic logon would defeat the purpose of even requiring passwords to unlock computers.
+
+**User experience impact**
+
+* Laptops lock after 20 minutes of inactivity. To voluntarily pause this, a [hot corner](https://support.apple.com/en-mo/guide/mac-help/mchlp3000/mac) can be configured to disable the screen saver. This is useful if you are, for example, watching an online meeting without moving the mouse and want to be sure the laptop will not lock.
+* Forgotten passwords can be fixed via MDM, instead of relying on potentially dangerous hints.
+* Guest accounts are not available.
+
+#### iCloud
+We do not apply ultra restrictive Data Loss Prevention style policies to our devices. Instead, by using different web-based tools we have, we ensure that the most critical company data is not stored on our laptops, so it can remain secure, while our laptops can remain productive.
```

Comment:
```suggestion
We do not apply ultra restrictive Data Loss Prevention style policies to our devices. Instead, by using our company Google Drive, we ensure that the most critical company data never reaches our laptops, so it can remain secure, while our laptops can remain productive.
```